jenkins/tests/test_codecov_analysis.py

```diff
@@ -22,9 +19,9 @@ def get_pulls(self, state, sort, direction):
         return self.prs
 
 
-class MockPR(object):
+class MockPR:
 
-    def __init__(self, title, commits=[], age=10000):
+    def __init__(self, title, commits=None, age=10000):
```

Comment:
Maybe just remove the default argument instead?  I think the new default would blow up in `reversed()` below, and we specify it in all calls anyway.